docs/04-reference/wingsdk-spec.md

```diff
@@ -292,22 +292,63 @@ resource Bucket {
 
   /**
    * Get a file from the bucket.
+   *
+   * @throws Will throw if the file doesn't exist.
    */
   inflight get(key: str): Blob;
 
+  /**
+   * Get a file from the bucket if it exists.
+   */
+  inflight try_get(key: str): Blob?;
+
   /**
    * Delete a file from the bucket.
+   *
+   * @throws Will throw if the file doesn't exist.
    */
   inflight delete(key: str): void;
 
+  /**
+   * Delete a file from the bucket if it exists.
+   */
+  inflight try_delete(key: str): bool;
+
+  /**
+   * Check if a file exists in the bucket.
+   */
+  inflight exists(key: str): bool;
+
+  /**
+   * Get the metadata of a file in the bucket.
+   */
+  inflight stats(key: str): FileMetadata;
+
+  /**
+   * Move a file to a new location in the bucket. If the destination file already
+   * exists, it will be overwritten.
+   *
+   * @throws Will throw if the `src` file doesn't exist.
+   */
+  inflight move(src: str, dst: str): void;
```

Comment:
What about calling this rename? When I think of move api it makes me think that I could specify the destination as a directory. move(“some_key”, “a/“)